src/main/pg/pg_ids.h

```diff
@@ -129,7 +129,8 @@
 #define PG_SDIO_CONFIG 532
 #define PG_DISPLAY_PORT_CRSF_CONFIG 533  // no longer required -- never released
 #define PG_TIMER_IO_CONFIG 534 // used to store the index for timer use in timerHardware array in target.c
-#define PG_BETAFLIGHT_END 534
+#define PG_RX_SPI_CONFIG 535
```

Comment:
If we add this parameter group, then it probably makes sense to move the SPI RX related parts of  `PG_RX_CONFIG` into it.

Reply:
There is a hierarchy of software and associated config data for each layer of:
- Receiver driver (`rxConfig`)
- Protocol driver (e.g., `rxFrSkySpiConfig`)
- Chip driver (e.g., cc2500 and nrf24)
- Chip common driver (`rxSpiConfig`)
- Bus driver

And yes, some refactoring is required, but I'm not sure if this is a time to do it.

I will, however, will change constant names for RF chip/bob pins that are prefixed by `FRSKY`; e.g., `RX_FRSKY_SPI_GDO_0_PIN` to `RXSPI_CC2500_GDO_0_PIN` or similar. (Similar for other `RX_FRSKY_SPI_` constants, which would probably be renamed to `RXSPI_`s.

Reply:
What about chip driver specific parameters? Go into `rxConfig`, too?

Reply:
Sorry, I think there was a misunderstanding.

In my opinion, parameter groups should be used in such a way that they align with how optional features are included in the build or not (where applicable). As for the chip / protocol specific parameters, I assume that in most cases there will only be one SPI RX chip on any given board, so it makes sense to be able to include each of the driver / protocol stacks individually, together with a parameter group holding its config.

My initial remark was only about `rx_spi_protocol`, `rx_spi_id`, and `rx_spi_rf_channel_count`, which are kept in `rxConfig` at the moment. They should not be in there, as 95% of the targets have no use for them, and if we are creating a parameter group specifically for SPI RX, they should be moved in there. I do not expect this to require much refactoring, since they are not used in many places.